app/grandchallenge/workstation_configs/models.py

```diff
@@ -144,8 +144,21 @@ class WorkstationConfig(TitleSlugDescriptionModel, UUIDModel):
         (SLAB_RENDER_METHOD_AVERAGE, "Average"),
     )
 
+    IMAGE_CONTEXT_DEFAULT = "DEF"
+    IMAGE_CONTEXT_PATHOLOGY = "PATH"
+    IMAGE_CONTEXT_OPTHAMOLOGY = "OPTHA"
+    IMAGE_CONTEXT_MPMRI = "MPMRI"
+
+    IMAGE_CONTEXT_CHOICES = (
+        (IMAGE_CONTEXT_DEFAULT, "Default"),
+        (IMAGE_CONTEXT_PATHOLOGY, "Pathology"),
+        (IMAGE_CONTEXT_OPTHAMOLOGY, "Opthamology"),
+        (IMAGE_CONTEXT_MPMRI, "Multiparametric MRI"),
+    )
```

Comment:
Django has a nicer way of defining these now with `TextChoices` or `IntegerChoices`. You can see it used here https://github.com/comic/grand-challenge.org/blob/51a2392a137411e8f7d84c0c6b5b7dc87c9f0041/app/grandchallenge/workstations/models.py#L282-L309

Documentation here https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types

This is preferred for new choice options, the old stuff is being refactored whenever I touch it.

Reply:
Came across this, but thought I'd keep everything in the style that was chosen. Will refactor.